ovirtlago/virt.py

```diff
@@ -76,6 +77,31 @@ def engine_vm(self):
     def host_vms(self):
         return self._host_vms[:]
 
+    def get_ovirt_cpu_family(self):
+        ovirt_cpu_families = {
```

Comment:
This might be nicer as a constant, that way anyone can check the supported ones by introspection (it would be nice also to have a way to notify if the cpu is not in the supported list)